website/src/pages/en/substreams/sps/introduction.mdx

```diff
@@ -5,6 +5,12 @@ sidebarTitle: Introduction
 
 Boost your Subgraph's efficiency and scalability by using [Substreams](/substreams/introduction/) to stream pre-indexed blockchain data.
 
+## Prerequisites
```

Comment:
Are these really "Prerequisites"? I feel like "Disclaimers" would be more appropriate?